src/utils/serverRequests.js

```diff
@@ -152,3 +152,65 @@ export const removeAllWatchHistory = async(dispatch) => {
       console.log(err);
   }
 }
+
+export const addNewPlaylist = async(playlistTitle, dispatch) => {
```

Comment:
Appreciation: Good that you have put all server requests in a single file.

Reply:
Thanks